### PR TITLE
feat: implemented button

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    flex-flow: row wrap;
+    align-items: center;
+    padding: 1rem;
+    border: 1px solid;
+    border-radius: 10px;
+    gap:15px;
+    margin-top: 1rem;
+}
+button {
+    padding: 0.5rem 2rem;
```
Comment (MaruthiPothuganti) [inline Components/Button/button.css]: Avoid styling using tag names

File: Components/Button/button.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button</title>
```
Comment (MaruthiPothuganti) [inline Components/Button/button.html]: Use title as Button | Ozone UI for consistency

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes. Everything else looks good.

File: Components/Button/button.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Button/button.css]: Sure,Thanks

File: Components/Button/button.html (hunk 2 above)
Comment (govardhanreddyc79) [reply Components/Button/button.html]: okay

**Review by saicharan-gouru — APPROVED** [review]
-Please add screenshot or gif for reference
-Code is structured good.
-Good use of naming conventions

**Comment by govardhanreddyc79** [comment]
Button Implemented

**Pull request merged**